content/videos/tipsandtricks-k8s-autodiscovery.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Datadog Tips & Tricks - Datadog Autodiscovery on Kubernetes
+kind: video
+language: en
+wistiaid: ypvryxgc2c
+videos:
+    - Datadog Tips & Tricks
+    - Introduction
+    - Kubernetes
+    - Autodiscovery
+    
+summary: Orchestration platforms like Docker Swarm, Kubernetes, and Amazon ECS make running containerized services easier and more resilient by handling workload scheduling and request routing. But all of that makes monitoring more difficult. How can you reliably monitor a service which by design constantly changes the definition normal as it moves applications between hosts? The Datadog Agent can automatically track which services are running where, thanks to its Autodiscovery feature. This video will show you how easy it is to get started with Autodiscovery on Kubernetes with Datadog.
+---
+Orchestration platforms like Docker Swarm, Kubernetes, and Amazon ECS make running containerized services easier and more resilient by handling workload scheduling and request routing. But all of that makes monitoring more difficult. How can you reliably monitor a service which by design constantly changes the definition normal as it moves applications between hosts? The Datadog Agent can automatically track which services are running where, thanks to its Autodiscovery feature. This video will show you how easy it is to get started with Autodiscovery on Kubernetes with Datadog.
+
+* [Container Challenges](?wtime=5)
+* [Documentation](?wtime=36)
+* [Adding the Agent to Kubernetes](?wtime=49)
+* [Setting up the Guestbook Example](?wtime=77)
+* [Redis is Automatically Configured](?wtime=110)
+* [Configcheck](?wtime=128)
+* [Redis auto_conf](?wtime=149)
+* [Three Autodiscovery Scenarios](?wtime=178)
+* [Set Up Annotations](?wtime=210)
+* [Setting Up Multiple Configurations](?wtime=306)
+
+
+Sign up for a trial account here: [https://app.datadoghq.com/signup](https://app.datadoghq.com/signup)
```

Comment:
1) There's already a Free Trial link in the Docs site's header
2) Aside from that header link, we don't usually hawk the free trial in Docs 
3) If we still want to link here, why not have [here] be the link text?